app/uk/gov/hmrc/transitmovementsrouter/controllers/MessagesController.scala

```diff
@@ -164,4 +170,30 @@ class MessagesController @Inject() (
     case _                     => EitherT.leftT(PresentationError.badRequestError(s"${messageType.code} is not valid for requests"))
   }
 
+  private def extractMovementMessageId(sdesResponse: SdesNotificationItem): (MovementId, MessageId) =
+    ConversationId(
+      UUID.fromString(
+        sdesResponse.conversationId.get.value
+      )
+    ).toMovementAndMessageId
+
+  def handleSdesResponse() =
+    Action.async(parse.json) {
+      implicit request =>
+        implicit val hc: HeaderCarrier = HeaderCarrierConverter.fromRequest(request)
+        (for {
+          sdesResponse <- parseAndLogSdesResponse(request.body)
+          (movementId, messageId) = extractMovementMessageId(sdesResponse)
+          messageStatus = sdesResponse.notification match {
+            case SdesNotification.FileProcessed         => MessageStatus.Success
+            case SdesNotification.FileProcessingFailure => MessageStatus.Failed
+          }
+          persistenceResponse <- persistenceConnector
```

Comment:
This piece of code will get executed for FileReceived and FileReady scenarios as well. Also can not see logic to ignore for FileReceived and FileReady response from SDES?